packages/api/src/routes/medical/document.ts

```diff
@@ -134,4 +140,52 @@ router.get(
   })
 );
 
+/**
+ * GET /document/upload-url
+ *
+ * Returns a signed url to upload a file to S3.
+ *
+ * @param patientId - The patientId of the patient.
+ * @param organizationName - The name of the organization that created the document.
+ * @param practitionerName - The name of the practitioner that created the document.
+ * @param fileDescription - The description of the file.
+ *
+ * @return presigned url
```

Comment:
ditto: more context on what it contains (not just the url) and hint on how it should be used